test/behavioral/tests/tabs.ts

```diff
@@ -24,11 +26,11 @@ const cycleRight = async (driver: WebDriver) => {
 
 export default async (
     t: TestContext,
-    snappit: Snappit,
     driver: WebDriver,
+    tabID: string,
     tabName: string
 ) => {
-    const tab = await getTab(driver, tabName);
+    const tab = await getTab(driver, tabID);
     await tab.click();
     t.is(await tab.getText(), tabName);
     await cycleLeft(driver);
```

Comment:
_(outside current scope of work)_
Logic in this function doesn't seem to be doing what might be expected.

It looks like we're trying to test the LEFT/RIGHT key behavior, but we aren't testing if the active tab changes. All we're testing is that the selected tab text isn't modified as we cycle left/right.

Reply:
That's a good point. It should be fixed later though since this is old code.